process/analysis.py

```diff
@@ -99,27 +158,48 @@ def analysis(r):
         os.makedirs(r.config['region_dir'])
 
     archive_parameters(r, settings)
+    
+    """Conditional Google Earth Engine authentication """
+    try:

```

Comment:
Rather than alert user if GEE is not configured, it should just be assumed that the user does not wish to run GEE indicators (the default).   The way you handle this later is better I think, 
```
if r.config.get('gee', False):
  ...
```